libraries/core/src/main/kotlin/io/element/android/libraries/core/notifications/NotificationTroubleshootTest.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.core.notifications
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.StateFlow
+
+interface NotificationTroubleshootTest {
+    val order: Int
+    val state: StateFlow<NotificationTroubleshootTestState>
+    fun isRelevant(data: TestFilterData): Boolean = true
+    suspend fun run(coroutineScope: CoroutineScope)
+    suspend fun reset()
+    suspend fun quickFix(coroutineScope: CoroutineScope) {
+        error("Quick fix not implemented, you need to override this method in your test")
```

Comment:
I agree, but I guess the developer who add a test with a quick fix will test it and so see the error.
And since quick fix support is optional I prefer to provide a default implementation.